### PR TITLE
Update docs/RELEASE.md and NEWS

### PR DESCRIPTION
**Pull request by bluca**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
+
+        * Support for split-usr (/usr/ mounted separately during late boot,
+          instead of being mounted by the initrd before switching to the rootfs)
+          and unmerged-usr (parallel directories /bin/ and /usr/bin/, /lib/ and
+          /usr/lib/, …) has been removed. For more details, see:
+          https://lists.freedesktop.org/archives/systemd-devel/2022-September/048352.html
+
+        * We intend to remove cgroup v1 support from a systemd release after
+          the end of 2023. If you run services that make explicit use of
+          cgroup v1 features (i.e. the "legacy hierarchy" with separate
```
Comment (yuwata) [inline NEWS:15]: How about hybrid one?

File: NEWS [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+
+        * Support for the SystemdOptions EFI variable is deprecated.
+          'bootctl systemd-efi-options' will emit a warning when used. It seems
+          that this feature is little-used and it is better to use alternative
+          approaches like credentials and confexts. The plan is to drop support
+          altogether at a later point, but this might be revisited based on
+          user feedback.
+
+        * systemd-run's switch --expand-environment= which currently is disabled
+          by default when combined with --scope, will be changed in a future
```
Comment (yuwata) [inline NEWS:33]: `will be` -> `may be` ??

**Review by yuwata — APPROVED** [review]
LGTM. The comment below is not necessary to be addressed or answered at least now. Let's discuss later if necessary. Merging.

File: NEWS (hunk 2 above)
Comment (bluca) [reply NEWS:33]: pretty sure we want to do that, it's not hypothetical, but there's some client software to fix

**Pull request merged**